### ISSUE TITLE
Decreased accuracy of results

### ISSUE THREAD

**Issue by qingyunwudaoletu** [MarsTechHAN/keras2ncnn#37]
when i convert h5 model to ncnn model
     process with the training data,  the ncnn model result is worse than h5 result
I  check the input data and find nothing different

i want to checkout if  it is a problem with model conversion

but there is error with debug mode

![image](https://user-images.githubusercontent.com/31977541/146192910-6c49c316-0ce8-4506-9300-c67334957d10.png)

please help~

Thankyou very much

**Comment by qingyunwudaoletu** [comment]
the model Link
https://drive.google.com/file/d/1RyN9Ccfp1g8NJldr-CR7rr_UWAfYf9ln/view?usp=sharing

**Comment by MarsTechHAN** [comment]
See it, I will take a look, but in practical terms, it is pretty difficult to solve the misalignment of accuracy.

**Comment by qingyunwudaoletu** [comment]
please help me  to use the debug mode.

it is so cool to show the convert result

Give my best wish to you and your family

**Comment by MarsTechHAN** [comment]
I fix some critical bugs, can you try the newest 0.2.0 version？

**Comment by qingyunwudaoletu** [comment]
ok,thanks. it is solved by update the relu6 bug report
